build.sbt

```diff
@@ -66,11 +66,11 @@ lazy val math = crossProject(JVMPlatform, JSPlatform)
   )
 
 lazy val testkit = crossProject(JVMPlatform, JSPlatform)
-  .crossType(CrossType.Full)
+  .crossType(CrossType.Pure)
   .in(file("modules/testkit"))
-  .dependsOn(math)
+  .dependsOn(core)
   .settings(
-    name := "gsp-math-testkit",
+    name := "lucuma-testkit",
```

Comment:
Since there will be other lucuma projects, I guess this should be `lucuma-core-testkit` (and `lucuma-core-tests` below)?

Reply:
Good catch, agree.

Reply:
👍